styles.css

```diff
@@ -0,0 +1,365 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+nav {
+  background-color: #3e3c3c;
+  height: 92px;
+  position: absolute;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+nav .bar1,
+.bar2,
+.bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: white;
+  margin: 6px 0;
+}
+
+.name-container {
+  flex: 1 0 250px;
+}
+
+.name-container h2 {
+  color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.links-container ul {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.links-container {
+  display: none;
+  background-color: #3e3c3c;
+}
+
+.header {
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  background-image: url(assets/Illustration-header.png);
+  background-size: 40%;
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.header .job-desc {
+  font-family: "Roboto", sans-serif;
+  color: #fff5e1;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 32px;
+  margin-left: 10px;
+}
+
+.header .intro {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff5e1;
+  max-width: 50%;
+  margin-left: 10px;
+}
+
+.icons-container > ul {
+  display: flex;
+  justify-content: space-between;
+  max-width: 50%;
+  margin-left: 10px;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.recent-works {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  align-items: center;
+}
+
+.recent-works h2 {
+  margin-top: 30px;
+  font-family: "Crete Round", sans-serif;
+  color: #1c1a19;
+}
```

Comment:
- [ ] As requested by the previous code reviewer please 🙏🏽 use the correct font-styles for the title to match the Figma template 👇🏽 
- [ ] Add the correct margins between the title, the line, and the works section.

![image](https://user-images.githubusercontent.com/58642949/219782523-fb644567-9c2f-46dd-bfcb-da72414cee92.png)

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219783125-96377b0d-6275-4378-a474-c8e3550731ed.png) | ![image](https://user-images.githubusercontent.com/58642949/219783225-5a8dd8dd-a889-4ec1-9620-256329923df0.png) |